### PR TITLE
[CORE-1863] Allow for running without Java agents

### PR DESCRIPTION
**Pull request by woggioni**
added a Java system property to allow for launching a Flask jar without its embedded Java agents

### CODE REVIEW

File: flask/flask-launcher/src/main/java/net/corda/flask/launcher/Launcher.java [hunk 1]
```diff
@@ -126,7 +130,10 @@ final int launch(Manifest manifest, List<String> jvmArgs, List<String> javaAgent
             if(jvmArgs != null) {
                 builder.getJvmArgs().addAll(jvmArgs);
             }
-            if(javaAgents != null) {
+            boolean disableJavaAgents = Optional.ofNullable(
+                    System.getProperty(Flask.JvmProperties.NO_JAVA_AGENT))
+                    .map(Boolean::parseBoolean).orElse(false);
```
Comment (chrisr3) [inline flask/flask-launcher/src/main/java/net/corda/flask/launcher/Launcher.java:135]: What about using [`Boolean.getBoolean(NO_JAVA_AGENT)`](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-)?

**Review by chrisr3 — COMMENTED** [review]
Just a thought...

File: flask/flask-launcher/src/main/java/net/corda/flask/launcher/Launcher.java (hunk 1 above)
Comment (woggioni) [reply flask/flask-launcher/src/main/java/net/corda/flask/launcher/Launcher.java:135]: I wasn't aware of its existence

**Review by chrisr3 — APPROVED** [review]
LGTM

**Pull request merged**